lr_face/data.py

```diff
@@ -880,3 +952,18 @@ def split_by_identity(
         data = data.images
     train_idx, test_idx = next(gss.split(data, groups=identities))
     return [data[idx] for idx in train_idx], [data[idx] for idx in test_idx]
+
+
+def get_benchmark_images():
+    """creates a fixed set of images to compute quality scores against.
+
+    The images are taken from LFW and increasingle reduced in resolution"""
+
+    # TODO save this as standalone set when we settle on what images to use?
+    data = LfwDevDataset(True)
+    # TODO resolution reduction? somewhat involved for FaceImages
+    images = data.images
+    return images[:100]
+
+
+BENCHMARK_IMAGES: List[FaceImage] = get_benchmark_images()
```

Comment:
why do this here instead of in def quality_score(self)?

Reply:
maar 1x uitrekenen